LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Microsoft Corporation
+Copyright (c) 2018 - present Microsoft Corporation
```

Comment:
Per guidance from our legal team, all years should instead be removed to avoid constant updates. Could you modify this PR to instead remove 2018?

Reply:
I seem to have deleted the repository, therefore this is an orphaned pull request. I have opened another one at #6595, sorry for the inconvenience, I thought this pull request had been merged.